test/UnitTests/TestFramework.UnitTests/TestFramework.UnitTests.csproj

```diff
@@ -4,7 +4,7 @@
     <NetStandardNetFrameworkHolder>net48</NetStandardNetFrameworkHolder>
     <TargetFrameworks>$(NetStandardNetFrameworkHolder);net8.0;net9.0</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);$(WinUiMinimum)</TargetFrameworks>
-    <IsNetCoreApp Condition=" '$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'net9.0' OR '$(TargetFramework)' == '$(WinUiMinimum)' ">true</IsNetCoreApp>
+    <IsNetCoreApp Condition=" $([System.String]::Copy('$(SupportedNetFrameworks)').Contains('$(TargetFramework)')) OR '$(TargetFramework)' == '$(WinUiMinimum)' ">true</IsNetCoreApp>
```

Comment:
`$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'` might be a better option.